index.html

```diff
@@ -265,10 +265,10 @@ <h4 class="my-3">Ventures</h4>
               <img class="mx-auto rounded-circle" src="assets/img/team/current/principal.jpg" alt="..." />
               <h4>Dr. S Brilly Sangeetha</h4>
               <p class="text-muted">Principal, IESCE</p>
-              <a class="btn btn-dark btn-social mx-2" href="https://linkedin.com/in/brilly-sangeetha-5b6a6a122/"><i
+              <a class="btn btn-dark btn-social mx-2" href="https://linkedin.com/in/brilly-sangeetha-5b6a6a122/" rel="noopener noreferrer" target="_blank"><i
```

Comment:
LinkedIn id changed, its [`https://www.linkedin.com/in/dr-s-brilly-sangeetha-5b6a6a122/`](https://www.linkedin.com/in/dr-s-brilly-sangeetha-5b6a6a122/) now.

Reply:
changes added